<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>over</class>
 <widget class="QWidget" name="over">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vampire Survivor</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/resourses/background.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>1520</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(76,79,113);
border:5px solid rgba(196,156,89);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>战斗数据</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="role">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/role1.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>1110</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(45,63,197);
border-radius:25px;
color:white;
border:10px solid rgba(196,156,89);</string>
   </property>
   <property name="text">
    <string>完 成</string>
   </property>
  </widget>
  <widget class="QLabel" name="map1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>关卡：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="map2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>280</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="time2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>370</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="time1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>存活时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kill2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kill1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>击杀数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gain1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>获取金币：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gain2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>550</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
